app/views/pages/register.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-md-8 text-center">
+      <h1 class="display-4">Register Yourself</h1>
+
+      <select id="registration-option" class="form-select mb-4" onchange="handleSelectChange(this)">
+        <option value="">Choose an option</option>
+        <option value="<%= new_company_admin_registration_path %>">Register as Company Admin</option>
+      </select>
+    </div>
+  </div>
+</div>
+
+<script>
```

Comment:
remove this script tags and explore turbo as well as js Pipeline in rails 
try to user it through turbo or add relative js controller